test/common/index.js

```diff
@@ -746,3 +749,47 @@ exports.getTTYfd = function getTTYfd() {
   }
   return tty_fd;
 };
+
+// exports.tag*: tag functions for tagged template literals
+
+// Removes all literal line breaks and leading spaces
+exports.tagGlue = function tagGlue(strings, ...expressions) {
+  const breaks = /\n */g;
```

Comment:
I would prefer to keep them self-sufficient while they are not used in libs critical for performance.